### PR TITLE
Fixed Velocity test. Removed exceptions from Meteor callbacks, as discussed.

### PR DESCRIPTION
**Pull request by larspetrus**

### CODE REVIEW

**Comment by jfly** [comment]
I thought we agreed to `console.error()` this warnings instead of `throw`-ing. This looks like we're ignoring them entirely?

**Comment by larspetrus** [comment]
I forgot. Will put it in.

File: client/uploadScrambles.js [hunk 1]
```diff
@@ -105,12 +105,11 @@ function extractJsonFromZip(filename, zipId, pw, cb) {
         } else {
           cb("Wrong password");
         }
-        return;
-      } else {
-        throw error;
       }
+      console.error("Meteor.call() error: " + err);
```
Comment (jfly) [inline client/uploadScrambles.js]: It looks like the behavior here has changed. Won't this run the console.error(...) even when they just typed in an incorrect password?

Comment (larspetrus) [inline client/uploadScrambles.js]: Yes, I didn't see that the code made that distinction.

Though when I see it now, I actually prefer logging all errors. Makes the code simpler and is consistent.

Comment (jfly) [inline client/uploadScrambles.js]: I would like to log all errors. I don't think an incorrect password counts here, though, as we are notifying the user anyways.

Comment (larspetrus) [inline client/uploadScrambles.js]: We're both making "Let's not spend extra effort" arguments.

To minimize this effort I changed the code to your version.

**Comment by jfly** [comment]
Looks great, thanks!

**Pull request merged**